care/users/migrations/0009_alter_user_user_type.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 4.2.2 on 2023-09-09 17:02
+
+from django.db import migrations, models
+
+
+def forwards_func(apps, schema_editor):
+    user_model = apps.get_model("users", "User")
+    user_list = []
+    for user in user_model.objects.filter(user_type=10):
+        user.user_type = 12
+        user_list.append(user)
+
+    user_model.objects.bulk_update(user_list, ["user_type"])
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("users", "0008_rename_skill_and_add_new_20230817_1937"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="user",
+            name="user_type",
+            field=models.IntegerField(
+                choices=[
+                    (2, "Transportation"),
+                    (3, "Pharmacist"),
+                    (5, "Volunteer"),
+                    (9, "StaffReadOnly"),
+                    (10, "Staff"),
+                    (12, "Nurse"),
+                    (15, "Doctor"),
+                    (20, "Reserved"),
+                    (21, "WardAdmin"),
+                    (23, "LocalBodyAdmin"),
+                    (25, "DistrictLabAdmin"),
+                    (29, "DistrictReadOnlyAdmin"),
+                    (30, "DistrictAdmin"),
+                    (35, "StateLabAdmin"),
+                    (39, "StateReadOnlyAdmin"),
+                    (40, "StateAdmin"),
+                ]
+            ),
+        ),
+        migrations.RunPython(forwards_func),
```

Comment:
shouldn't we have a `reverse_code` to migrate all nurses back to staff?

cc: @sainak

Reply:
I don't think so. Staffs are already present. We have to change staff to nurse. There is no point of adding a function to migrate nurses back to staffs.